bindings/node/test/data/cmd.json

```diff
@@ -0,0 +1,6 @@
+{
+    "find": "test",
+    "filter": {
+        "ssn": "457-55-5462"
+    }
+}
```

Comment:
nit: these files should end with newlines per unix convention.

Reply:
:+1: